tests-functional/tests/test_activity_center_notifications.py

```diff
@@ -0,0 +1,190 @@
+        assert all(
+            (
+                response["result"]["activityCenterNotifications"][0]["id"] == message_id,
+                response["result"]["activityCenterNotifications"][0]["read"] is True,
+            )
+        )
+
+        result = self.sender.wakuext_service.rpc_request(
+            method="activityCenterNotifications", params=[{"cursor": "", "limit": 20, "activityTypes": [5]}]
+        ).json()["result"]
+        assert result["notifications"][0]["read"] is True
+
+        response = self.sender.wakuext_service.rpc_request(
+            method="markActivityCenterNotificationsUnread",
+            params=[
+                [
+                    message_id,
+                ],
+            ],
+        ).json()
+        self.sender.verify_json_schema(response, method="wakuext_markActivityCenterNotificationsUnread")
+        assert all(
+            (
+                response["result"]["activityCenterState"]["hasSeen"] is False,
+                response["result"]["activityCenterNotifications"][0]["id"] == message_id,
+                response["result"]["activityCenterNotifications"][0]["read"] is False,
+            )
+        )
+
+        result = self.sender.wakuext_service.rpc_request(
+            method="activityCenterNotifications", params=[{"cursor": "", "limit": 20, "activityTypes": [5]}]
+        ).json()["result"]
+        assert result["notifications"][0]["read"] is False
+
+    def test_accept_activity_center_notifications(self):
+        message_id = self.send_contact_request_and_wait_for_signal_to_be_received()
+        response = self.receiver.wakuext_service.rpc_request(
+            method="acceptActivityCenterNotifications",
+            params=[
+                [
+                    message_id,
+                ],
+            ],
+        ).json()
+        self.receiver.verify_json_schema(response, method="wakuext_acceptActivityCenterNotifications")
+
+        result = self.receiver.wakuext_service.rpc_request(
+            method="activityCenterNotifications", params=[{"cursor": "", "limit": 20, "activityTypes": [1, 2, 3, 4, 5, 7, 8, 9, 10, 23, 24]}]
+        ).json()["result"]
+        assert all((result["notifications"][0]["accepted"] is True, result["notifications"][0]["id"] == message_id))
+
+    def test_dismiss_activity_center_notifications(self):
+        message_id = self.send_contact_request_and_wait_for_signal_to_be_received()
+        response = self.receiver.wakuext_service.rpc_request(
+            method="dismissActivityCenterNotifications",
+            params=[
+                [
+                    message_id,
+                ],
+            ],
+        ).json()
+        self.receiver.verify_json_schema(response, method="wakuext_dismissActivityCenterNotifications")
+
+        result = self.receiver.wakuext_service.rpc_request(
+            method="activityCenterNotifications", params=[{"cursor": "", "limit": 20, "activityTypes": [1, 2, 3, 4, 5, 7, 8, 9, 10, 23, 24]}]
```

Comment:
It is reused, maybe we can put it to some internal method with explanation what it actually does?